deep_speech_2/deploy/ctc_decoders.cpp

```diff
@@ -0,0 +1,337 @@
+#include <iostream>
+#include <map>
+#include <algorithm>
+#include <utility>
+#include <cmath>
+#include <limits>
+#include "fst/fstlib.h"
+#include "ctc_decoders.h"
+#include "decoder_utils.h"
+#include "path_trie.h"
+#include "ThreadPool.h"
+
+std::string ctc_best_path_decoder(std::vector<std::vector<double> > probs_seq,
+                                  std::vector<std::string> vocabulary)
+{
+    // dimension check
+    int num_time_steps = probs_seq.size();
+    for (int i=0; i<num_time_steps; i++) {
```

Comment:
Please add a pre-commit hook for c++ coding style check.
`for (size_t i = 0; i < num_time_steps; i++) {`

Reply:
Done